cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/optimized/MessagesApiModel.kt

```diff
@@ -90,20 +90,37 @@ data class Campaigns(
 
 @Serializable
 data class CCPA(
-    @SerialName("applies") val applies: Boolean?,
-    @SerialName("consentedAll") val consentedAll: Boolean?,
-    @SerialName("dateCreated") override val dateCreated: String?,
-    @SerialName("message") override val message: JsonElement?,
-    @SerialName("messageMetaData") override val messageMetaData: MessageMetaData?,
-    @SerialName("newUser") val newUser: Boolean?,
-    @SerialName("rejectedAll") val rejectedAll: Boolean?,
-    @SerialName("rejectedCategories") val rejectedCategories: List<String>?,
-    @SerialName("rejectedVendors") val rejectedVendors: List<String>?,
-    @SerialName("signedLspa") val signedLspa: Boolean?,
-    @Serializable(with = CcpaStatusSerializer::class) val status: CcpaStatus?,
-    @Serializable(with = CampaignTypeSerializer::class) override val type: CampaignType,
-    @SerialName("url") override val url: String?,
-    @SerialName("webConsentPayload") val webConsentPayload: JsonObject?,
+    @SerialName("applies")
```

Comment:
If I were you, I would put a `@SerialName` tag and value declaration in the same line. It's kinda easier to read for me. But that's 100% optional :) Is it against kotlin code style?

Reply:
well, actually from my pov it is better not only to move those to a new line, but also separate one line between parameters, to make it more visible, but I will do that when I'm going to separate those models in the future

Kotlin does not have a strict code style for that, so again, it is only the matter of taste :)